bundles/SyliusThemeBundle/your_first_theme.rst

```diff
@@ -0,0 +1,140 @@
+Your first theme
+================
+
+Themes location and definition
+------------------------------
+
+Private themes should be added to ``app/themes`` directory by default. Every theme should have a default confguration
+located in ``composer.json`` file. The only required parameter is ``name``, but it is worth to define other.
+
+.. code-block:: json
+
+    {
+      "name": "vendor/default-theme"
+    }
+
+When adding or removing theme, it's necessary to rebuild the container (same as adding new translation files in Symfony2) by clearing the cache (`app/console cache:clear`).
```

Comment:
Why exactly we need to do that? There are some new services created per theme? We would like use that bundle as a base for our theming system but in saas app clearing cache on every new theme installation is not an option ;/